llvm/lib/Target/SPIRV/SPIRVPreLegalizer.cpp

```diff
@@ -587,6 +587,40 @@ static void processSwitches(MachineFunction &MF, SPIRVGlobalRegistry *GR,
   }
 }
 
+static bool isImplicitFallthrough(MachineBasicBlock &MBB) {
+  if (MBB.empty())
+    return true;
+
+  // Branching spirv intrinsics are not detected by this generic method.
```

Comment:
We usually use SPIR-V (or SPIRV) in comments.

Reply:
Ah right, thanks! Fixed both occurences.